### PR TITLE
update README.md file

### PR DESCRIPTION
**Pull request by AugustasV**
Adding my name to Contributors section at README.md file

### CODE REVIEW

**Comment by filfreire** [comment]
@AugustasV thank you for your contribution. We'll merge this PR once you follow-up on the review of PR #26

**Review by filipemcarvalho — APPROVED** [review]
To be merged once #26 is merged

**Pull request merged**
